core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.controller.ts

```diff
@@ -100,6 +146,94 @@ angular.module('oppia').controller('TranslationModalController', [
       $scope.activeWrittenTranslation.html = '';
     };
 
+    $scope.getElementTexts = function(elements, type) {
+      const textWrapperLength = 6;
+      const attributes = Array.from(elements, function(element: HTMLElement) {
+        if (element.localName === 'oppia-noninteractive-image') {
```

Comment:
Can you add a comment here with an example HTML element? Thinking of a future reader, it would be confusing as to why we need to parse the HTML attribute this way.